pallets/sp-mvm/src/balance.rs

```diff
@@ -35,6 +43,11 @@ where
         address: &move_core_types::account_address::AccountAddress,
         ticker: &str,
     ) -> Option<VmBalance> {
+        if !is_ticker_supported(ticker) {
+            trace!("native balance ticker '{}' not supported", ticker);
+            return None;
```

Comment:
let's return error here? otherwise could be misunderstood, no?

Reply:
There's no error yet. VM doesn't require Result there now. So `Option::None` only.